cfme/utils/providers.py

```diff
@@ -127,7 +127,15 @@ def _filter_required_tags(self, provider):
 
     def _filter_required_flags(self, provider):
         """ Filters by required yaml flags """
+        required_test_flags = provider.data.get('required_test_flags', '')
+        if isinstance(required_test_flags, str):
+            required_test_flags = required_test_flags.split(',')
+        required_test_flags = [flag.strip() for flag in required_test_flags]
+
         if self.required_flags is None:
+            if required_test_flags is None:
```

Comment:
Ugh... At first glance I though this branch doesn't make sense to have. But I realized there is some ternary logic going on in the filters `(Flase != None)`. Then this is probably fine.